src/image/image.js

```diff
@@ -4,18 +4,19 @@
  * @for p5
  * @requires core
  */
-define(function (require) {
 
+// TODO: This IIFE is to avoid a false-positive global warning from jshint.
+// There's absolutely a better way to do this.
+(function() {
```

Comment:
This is because JSHint is complaining about the redefinition of "frames", which isn't technically valid—this logic isn't actually global because Browserify wraps it in a module, JSHint just doesn't know that. I have an open question out to the JSHint maintainers about how we can best suppress the error, after which we can remove this IIFE.